stdlib/public/core/MemoryLayout.swift

```diff
@@ -156,10 +163,13 @@ extension MemoryLayout {
   /// - Returns: The default memory alignment, in bytes, of the given value's
   ///   type. This value is always positive.
   @_transparent
-  public static func alignment(ofValue value: T) -> Int {
+  @_preInverseGenerics
+  public static func alignment(ofValue value: borrowing T) -> Int {
     return MemoryLayout.alignment
   }
+}
 
+extension MemoryLayout {
```

Comment:
Are we going to commit with the implicit defaults? When are going to be explicit everywhere?

Reply:
Yes, we'll be deliberately relying on implicit defaults in the stdlib! For now at least. 

We can introduce making things explicit as a coding convention later, but with `~Escapable` just around the corner, I'm not yet sure we'll actually like that.